### PR TITLE
blog: add update to V8 5.4 post

### PR DESCRIPTION
**Pull request by targos**
This is something I wrote before the v7 release but I forgot about it.
It's the first time I try to contribute a blog post. Please tell me if I'm doing something wrong!

### CODE REVIEW

File: locale/en/blog/community/update-v8-5.4.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+author: Michaël Zasso
+---
+
+With the release of Node.js 7.0.0, the V8 JavaScript engine has been upgraded from 5.1
+to its latest stable version, 5.4.
+It brings in new language features and increased performance.
+
+## New ECMAScript features
+
+### Exponentiation operator (ES216)
```
Comment (lpinca) [inline locale/en/blog/community/update-v8-5.4.md]: s/ES216/ES2016/

File: locale/en/blog/community/update-v8-5.4.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+const descriptors = Object.getOwnPropertyDescriptors(obj);
+/*
+{ x: {value: 0, writable: true, enumerable: true, configurable: true},
+  y: {value: 100, writable: true, enumerable: true, configurable: true} }
+*/
+```
+
+## Performance and memory optimizations
+
+### From [V8 5.2](http://v8project.blogspot.ch/2016/06/release-52.html)
```
Comment (lpinca) [inline locale/en/blog/community/update-v8-5.4.md]: Feel free to ignore but what do you think about using https in these links?

**Review by lpinca — APPROVED** [review]
LGTM with ES216 typo fixed.

File: locale/en/blog/community/update-v8-5.4.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+{ x: {value: 0, writable: true, enumerable: true, configurable: true},
+  y: {value: 100, writable: true, enumerable: true, configurable: true} }
+*/
+```
+
+## Performance and memory optimizations
+
+### From [V8 5.2](https://v8project.blogspot.ch/2016/06/release-52.html)
+
+Improvement of JavaScript built-ins, including `Array` operations like the `isArray` method, the `in` operator, and `Function.prototype.bind`.
```
Comment (ryanmurakami) [inline locale/en/blog/community/update-v8-5.4.md]: Do you want to make this bulleted to match the below sections, even though it's just one statement? I think that would flow better.

Comment (targos) [reply locale/en/blog/community/update-v8-5.4.md]: done

**Pull request merged**